tests/python/gaia-ui-tests/gaiatest/visual_components/switch.py

```diff
@@ -0,0 +1,18 @@
+from marionette_driver import Wait
+from gaiatest.visual_components.visual_component import VisualComponent
+
+
+class Switch(VisualComponent):
```

Comment:
ah, awesome! yeah, we might as well support both, although in current PR you are not actually using gaia-switch, it would be easier for the next person to apply it when gaia-switch needs to be triggered.